pom.xml

```diff
@@ -28,8 +28,15 @@
     </scm>
     <properties>
         <java.version>23</java.version>
```

Comment:
It is better to use LTS version in production

Reply:
I tried to use 23.0.1 and the latest versions but the project build returned the error:
"Fatal error compiling: error: release version 23.0.1 not supported"
or
"Fatal error compiling: error: release version last not supported"